content/chapter-03/SystemMetricsLab.md

```diff
@@ -1 +1,40 @@
-# Lab: common tools for system metrics
\ No newline at end of file
+# Lab: common tools for system metrics
```

Comment:
# Lab: Tools for Measuring Resource Utilization

Reply:
Done